### ISSUE TITLE
Test: accept a notification's primary action via keyboard

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#177672]
Refs #175140

- [x] Windows @roblourens 
- [x] macOS @hbons 
- [x] Linux @connor4312 

Complexity: 3

Authors: @meganroge

[Create Issue](https://github.com/microsoft/vscode/issues/new?body=Testing+%23177672%0A%0A&assignees=meganrogge,meganroge)

---

Historically, when screen reader users heard a notification, they'd have to navigate away from their current context to accept / interact with it. 

To improve this experience, when in screen reader mode, we've added `notification.acceptPrimaryAction` triggered via `KeyMod.CtrlCmd | KeyMod.Shift | KeyCode.KeyA` to accept a notification's primary action. If there are multiple notifications, they will be accepted sequentially. A notification's primary action appears in blue in the first position. 

Verify this works and your current context is unchanged.

For example:

1. focus in the editor
2. trigger a notification 
3. accept the notification via the keybinding
4. your focus should still be in the editor (repeat with focus in the terminal)

**Comment by hbons** [comment]
How can I trigger a notification?

**Comment by hbons** [comment]
Thanks @meganrogge. Turns out there's a[ Notification Tester](https://marketplace.visualstudio.com/items?itemName=svipas.notification-tester) extension, so I've used that.

On macOS with VoiceOver enabled (and VS Code in "Screen Reader Optimized" mode) I can confirm I can dismiss notifications using Cmd+Shift+A and focus stays in the editor. This also works with multiple notifications and I can dismiss them one at a time.
